### DISCUSSION TITLE
VFS Appointment Bot Discussions!

### DISCUSSION THREAD

**Discussion by ranjan-mohanty** [ranjan-mohanty/vfs-appointment-bot#49] [General]
## 👋 Welcome!
This forum is for open conversations about our project! Share ideas, ask questions, request new features, and discuss any issues you encounter. Let's collaborate and build something great together!

**Comment by Saikal87** [comment]
Dear Ranjan Mohanty

Would you please contact me on telegramm or please check your mail box

**Comment by amira-nemdil** [comment]
Dear Ranjan Mohanty,
Firstly, I want to thank you for your hard work and it's much appreciated as a cs student beginner
I have a request, can you please add ALGERIA > GERMANY 
if it's simple to edit the script can you help me by providing me with more steps? thank you appreciated!

**Reply by ranjan-mohanty** [reply]
https://github.com/ranjan-mohanty/vfs-appointment-bot/issues/58

**Comment by p42rthicle** [comment]
What do I put the parameters as? The center and the visa type and subtype?
Seems like doesnt accept what I type in simple string

**Comment by Rashidov21** [comment]
> ## 👋 Welcome!
> This forum is for open conversations about our project! Share ideas, ask questions, request new features, and discuss any issues you encounter. Let's collaborate and build something great together!


Dear Ranjan Mohanty I need a consultation on writing a bot like this for citizens of Uzbekistan and Tajikistan, Latvia, Lithuania, we can agree on the price of the service, can you contact me via email?
rashidovabdurakhmon@gmail.com

**Comment by visazxpert** [comment]
Hello sir, appreciate your skills ...can you add feature for https://visa.vfsglobal.com/are/en/ita/login
